src/web/theme/modules/Header/Navigation/Menu/Menu.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "theme/ui/atoms/Button";
+import Icon from "theme/ui/atoms/Icon";
+import { withState, withHandlers } from "recompose";
+import compose from "recompose/compose";
+
+const Menu = ({
+  setNavigationStep,
+  navigationSteps,
+  setOpenStateMenu,
+  isMenuOpened
+}) => {
+  const menuClasses = `menu ${isMenuOpened ? `open` : `close`}`;
+  return (
+    <div className={menuClasses}>
+      <div className="menu__container">
+        {navigationSteps.map(
+          step =>
+            step.selected ? (
+              <div key={step.name} className="menu__navButton selected">
```

Comment:
Same here about BEM usage.

Could we improve the documentation with BEM instructions to help?